staker/staker.gno

```diff
@@ -208,14 +219,14 @@ func GetAverageDelegated(delegate std.Address, start, end uint64) (*u256.Uint, e
 }
 
 // GetAverageDelegatedOverLast gets the average amount delegated over the last period seconds.
-func GetAverageDelegatedOverLast(delegate std.Address, period uint64) (*u256.Uint, error) {
+func GetAverageDelegatedOverLast(delegate string, period uint64) (*u256.Uint, error) {
```

Comment:
```suggestion
func GetAverageDelegatedOverLast(delegate string, period uint64) (string, error) {
```

FYI, gno http rpc service can not parse pointer return value.
How about we return string??

Reply:
changed all the functions to returning string types
https://github.com/gnoswap-labs/governance/pull/14/commits/6fd92005e9110f1ba48fbcf35e2d2eb2bbd19aa0